servicetalk-examples/docs/modules/ROOT/pages/grpc/index.adoc

```diff
@@ -80,9 +80,23 @@ xref:{page-version}@servicetalk::programming-paradigms.adoc#asynchronous-and-str
 
 Extends the async "Hello World" example to demonstrate compression of the response body.
 
-* link:{source-root}/servicetalk-examples/grpc/compression/src/main/java/io/servicetalk/examples/grpc/compression/async/CompressionExampleServer.java[CompressionExampleServer] - Waits for hello request from the client and responds with a compressed greeting response.
```

Comment:
@tkountis - fyi `async` isn't correct, not sure why our doc validation didn't check this.

Reply:
@bondolo ☝️ you are more familiar here.

Reply:
Yes, this change corrects the URL.